src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -903,12 +903,12 @@ all_docs_key_opts(#mrargs{keys = Keys, direction = Dir} = Args, Extra) ->
         Keys
     ).
 
-ad_skey_opts(#mrargs{start_key = SKey}) when is_binary(SKey) ->
+ad_skey_opts(#mrargs{start_key = SKey}) when SKey =/= undefined ->
     [{start_key, SKey}];
 ad_skey_opts(#mrargs{start_key_docid = SKeyDocId}) ->
     [{start_key, SKeyDocId}].
```

Comment:
We don't allow non-string document IDs. At first it would seem we should strongly validate here and allow only `binary|undefined` However, with start/end keys users may use lowest/highest values that are outside the range, something like `start_key=null`, `end_key=[{}]` or something similar and we wouldn't want to prevent them from doing it so this implementation would work well there.